tests/Integration/Behaviour/Features/Context/Domain/ModuleFeatureContext.php

```diff
@@ -118,6 +130,35 @@ public function updateModuleStatus(string $action, string $technicalName): void
         Module::resetStaticCache();
     }
 
+    /**
+    * @When /^I uninstall module "(.+)" with deleteFile (true|false)$/
+    */
+    public function uninstallModule(string $module, string $deleteFile): void
+    {
+ 
+        $this->getQueryBus()->handle(new UninstallModuleCommand($module, $deleteFile == "true"));
```

Comment:
If you want to check the `ModuleNotInstalledException` via the step above it needs to be caught first So you should wrap this code into a try/catch, catch a generic `ModuleException` (base class for all exceptions of the module domain) and "store" it using `$this->setLastException($e)` so it can be asserted in the next step

Similar to what was done in the `resetModule` method above